### PR TITLE
Edit user current status

### PR DESCRIPTION
**Pull request by ace0625**
- Edit user current status data in User list and User Profile

### CODE REVIEW

File: src/components/MyPage.js [hunk 1]
```diff
@@ -77,6 +58,49 @@ class MyPage extends Component {
     UserUtil.getRequestSetting(this.onRequestCallback.bind(this));
   }
 
+  getProfileImage(status) {
```
Comment (MechanicKim) [inline src/components/MyPage.js:61]: It is possible not to use 'image';
`return { uri: status.profile_picture };`
and
`return require('../resources/pattern.png');`

File: src/components/MyPage.js [hunk 2]
```diff
@@ -77,6 +58,49 @@ class MyPage extends Component {
...
+      if (status.experience[0].position) {
+        currentTask = status.experience[0].position.name;
+      } else {
+        return location;
+      }
+
+      return currentTask + ' at ' + location;
+    }  else if (status.education.length > 0) {
+      let lastIndex = status.education.length - 1;
+      let education = status.education[lastIndex];
```
Comment (MechanicKim) [inline src/components/MyPage.js]: It is good to use 'const'.
let lastIndex -> const lastIndex
let education -> const lastIndex

You can refer this URL
http://eslint.org/docs/rules/prefer-const.html

File: src/components/UserList/Row.js [hunk 3]
```diff
@@ -52,6 +52,20 @@ class Row extends Component {
       }
 
       return currentTask + ' at ' + location;
+    } else if (this.props.dataSource.education.length > 0) {
+      let lastIndex = this.props.dataSource.education.length - 1;
+      let education = this.props.dataSource.education[lastIndex];
```
Comment (MechanicKim) [inline src/components/UserList/Row.js]: You can refer this URL
http://eslint.org/docs/rules/prefer-const.html

Comment (MechanicKim) [reply src/components/UserList/Row.js]: Oh! @ace0625 my mistake!
This is same.
let -> const ^^;

Comment (ace0625) [reply src/components/UserList/Row.js]: Done!

File: src/components/MyPage.js (hunk 1 above)
Comment (ace0625) [reply src/components/MyPage.js:61]: Done!

**Comment by MechanicKim** [comment]
I reviewed it.
I have no comment any more.

LGTM

[Approved with PullApprove](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/112/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

**Pull request merged**